voting-web-views/src/styles/styles.scss

```diff
@@ -9,7 +9,7 @@ header,
 /*Intro*/
 
 .view-fullscreen {
-    background: url("https://mdbootstrap.com/img/Photos/Horizontal/People/12-col/img%20(61).jpg")no-repeat center center fixed;
+    background: url("http://greatnewsmag.com/wp-content/uploads/2015/12/vote_CCRI.jpg")no-repeat center center fixed;
```

Comment:
I strongly recommend you to not do this, as the image could get removed from the website at any time. Try to download it, and make it available offline